### DISCUSSION TITLE
Scarlett 2i2 2nd gen

### DISCUSSION THREAD

**Discussion by atman30** [geoffreybennett/alsa-scarlett-gui#127] [Q&A] [answered]
Is there any way to use alsa-scarlett-gui with my Scarlett 2i2 2nd gen?

**Comment by geoffreybennett** [comment, marked as answer]
Hi @atman30,

No there's not, not because of any lack in the software, but because it's irrelevant. The Scarlett 2i2 2nd Gen has no software controls; it's all switches and knobs on the front panel.

Regards,
Geoffrey.

**Reply by atman30** [reply]
Ok, thank you for the answer. It makes sense. 

Regards
Luca
